modules/income_and_assets/lib/income_and_assets/helpers.rb

```diff
@@ -32,7 +32,7 @@ def format_date_to_mm_dd_yyyy(date_string)
     # @return [Hash]
     #
     def split_currency_amount_sm(amount, field_lengths = {})
-      return {} if !amount || amount.negative? || amount >= 100_000
+      return {} if !amount || amount.negative? || amount >= 1_000_000
```

Comment:
I assume that the max value here is supposed to be 999,999.99 instead of 99,999.99

Reply:
If 99,999 is the max value and valid, then I think you want to allow values up to and including it.

`return {} if !amount || amount.negative? || amount > 99_999`

Reply:
Whoops, off by 100,000 ha

Reply:
Ok if that's the case, then I think it should be

`return {} if !amount || amount.negative? || amount > 999_999`

Reply:
I think you have to account for the cents as well (.99). This change is in multiple other outstanding PRs as well so honestly unless there is a functional change that needs to be made I’d rather leave it as is so I don’t have to update the other PRs

Reply:
Ah ok, the cents definitely make a difference here